### DISCUSSION TITLE
Another option in "nav_fw_launch_manual_throttle" for AFTER the launch is detected.

### DISCUSSION THREAD

**Discussion by Thieu-s** [iNavFlight/inav#8833] [Ideas]
I would like to have another option in "nav_fw_launch_manual_throttle" to go to this manual throttle AFTER the launch is detected.
So when you arm and raise the throttle, it will go to "nav_fw_launch_idle_thr", then after the launch is detected, instead of the value set in "nav_fw_launch_thr", it will go to "manual throttle".

So something like:
nav_fw_launch_manual_throttle = OFF
nav_fw_launch_manual_throttle = ON
nav_fw_launch_manual_throttle = LAUNCH

Depending on the wind, sometimes I want to launch at full throttle, but on calm days I like to launch at 50%

**Comment by Jetrell** [comment]
When I use `nav_fw_launch_manual_throttle = ON`. I find its greatest advantage, is being able to throw the plane at full throttle for the first couple of seconds, to get its mass moving. Then I back the throttle down to around 35%, once it has momentum, to conserve energy for the rest of the climb.
This energy saving is considerable, compared to the normal launch method.

**Reply by Thieu-s** [reply]
Exactly, but I like to throw the wing with the throttle at the idle throttle and the delays like the normal autolaunch procedure.
